### DISCUSSION TITLE
Low Earnings?

### DISCUSSION THREAD

**Discussion by jokinvipo** [engageub/InternetIncome#114] [Q&A]
Hello,

My apps are a giving very low earnings specially in HoneyGain.

All my proxies are GB/s speed in US region

![image](https://github.com/engageub/InternetIncome/assets/35331883/aa57390e-1840-42ae-877b-1b7e11cbd9bb)

This is normal or somehow they reduced alot my IP Traffic?

**Comment by engageub** [comment]
Hi,
It depends on IP quality and demand in the region. 
As long as the app is running and receiving traffic, the script is working as expected.
Official docker image is being used. 
Thank you

**Reply by engageub** [reply]
Hi,
Could you please read the recently updated [FAQ](https://github.com/engageub/InternetIncome?tab=readme-ov-file#-why-are-some-of-the-containers-for-the-same-application-offline) and make sure there are no errors related to DNS queries.

Thank you

**Reply by superit63** [reply]
Hi [jokinvipo](https://github.com/jokinvipo),

As far as I know, the earning almost depends on the IP quality score and location. The InternetIncome script is perfect to use at this time.

Thanks
